src/main/webapp/resources/js/pages/projects/samples/services/constants.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Initial state of the sample table
+ * @type {string}
+ */
+export const INITIAL_TABLE_STATE = JSON.stringify({
+  filters: { associated: null },
+  pagination: {
```

Comment:
Need to update the project samples table pagination to use the default pagination we had set up to keep the tables consistent. `getPaginationOptions` in `antdesign-table-utilities.js`

Reply:
This will actually break my table.  It will cause unnecessary table reloads.  The reason for this is that changing any of the pagination options triggers a table reload, therefor if you filter and there is a new total it will reload the table a second time thinking that the user updated something in the pagination and it needs to get new data.

Reply:
@deepsidhu85 me and @joshsadam did some testing and are able to use the call without any extra renders. He also updated all calls to `getPaginationOptions` to be done on table component initialization instead of using `useMemo` since this function is not computationally intensive so no benefit to using `useMemo`.

Reply:
Awesome thanks @ericenns